DEV-8/src/MatrixMultiplier.java

```diff
@@ -0,0 +1,34 @@
+import java.math.BigDecimal;
+import java.math.MathContext;
+
+/**
+ * Multiply matrixes
+ */
+public class MatrixMultiplier {
```

Comment:
Class which has only one method and no fields is a bad idea, this method has to be method of Matrix